bin/MadGraph5_aMCatNLO/cards/production/13TeV/tH_t_topdecay_LH_narrow_M1100/tH_t_topdecay_LH_narrow_M1100_proc_card.dat

```diff
@@ -0,0 +1,41 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.2.1                 2014-09-25         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model STP_UFO_freeWidth
+define Topo = t23 t23~
+define bb = b b~
+define WW = w+ w-
+define tt = t t~
+define ferm =  ve vm vt ve~ vm~ vt~ u c d s b u~ c~ d~ s~ b~ e- mu- ta- e+ mu+ ta+
+generate p p > j tt Topo  / h ,  tt > bb WW , ( Topo > h tt / z w+ w- ,  tt > bb WW )
```

Comment:
No,
I am asking if the width is actually reflected in the kinematics of the events after running them through pythia.  What I am afraid happens is that madgpraph produces the W's exactly on shell, and then pythia leaves them exactly on shell when doing the decays.  This should be checked, and if you want to use pythia for the decays then there may be a setting which needs to be changed in madgraph to sample properly the breit wigner of the W.  First step is to check what actually happens by running 100 or 1000 events through pythia and checking the distribution of the status 1 lv mass.